pages/gallery/gallery.js

```diff
@@ -0,0 +1,420 @@
+
+    <div class="my-2 text-h5">{{ $root.lang().gallery.category.mc_version }}</div>
+    <v-btn style="margin-bottom: 5px"  v-for="t in options.versions" v-on:click="updateRoute(t, 'version')" :key="t + $route.path" :class="{ 'mr-1': true, 'v-btn--active': t === version }" >{{ t }}</v-btn>
+
+    <div class="my-2 text-h5">{{ $root.lang().gallery.category.tags }}</div>
+    <v-btn style="margin-bottom: 5px"  v-for="t in options.tags" v-on:click="updateRoute(t, 'tag')" :key="t + $route.path" :class="{ 'mr-1': true, 'v-btn--active': t === tag }">{{ t }}</v-btn>
+
+    <div class="my-2 text-h5">{{ $root.lang().gallery.category.search }}</div>
+    <div class="my-2">
+      <v-text-field
+        v-model="current.search"
+        :append-outer-icon="current.search ? 'mdi-send' : undefined"
+        filled
+        clear-icon="mdi-close"
+        clearable
+        :placeholder="$root.lang().database.labels.search_texture"
+        type="text"
+        v-on:keyup.enter="startSearch"
+        @click:append-outer="startSearch"
+        @click:clear="clearSearch"
+      ></v-text-field>
+    </div>
+    
+    <v-list
+      two-line
+      style="padding: 10px;"
+    >
+      <template v-if="loading.status == true">
+        <div class="text-h6 py-6" style="padding: 0 10px !important">{{ loading.comments.length }}/{{ loading.steps }} {{ $root.lang().gallery.loading_message.general }}</div>
+        <span style="padding: 0 10px;">{{ loading.comments[loading.comments.length - 1] }}</span>
+      </template>
+      <template v-if="!loading.status && displayedTextures.length === 0">
+        <div class="text-h6 py-6" style="padding: 0 10px 10px !important">{{ $root.lang().global.no_results }}</div>
+      </template>
+
+      <div
+        v-if="!loading.status"
+        class="gallery-textures-container"
+      >
+        <div
+          v-for="(texture, index) in displayedTextures"
+          v-if="index <= displayedResults"
+          class="gallery-texture-in-container"
+          v-tooltip.right-start="{content: () => getAuthor(texture.textureID), html: true}"
+          v-on:click="openModal(texture.textureID)"
+        >
+          <img class="gallery-texture-image" onerror="this.onerror=null;this.src='https://database.compliancepack.net/images/bot/error.png';" :src="getTextureURL(texture.useID)" lazy-src="https://database.compliancepack.net/images/bot/loading.gif" />
+          <v-img class="gallery-texture-background" contain style="position: absolute; z-index: -1;" src="https://raw.githubusercontent.com/Compliance-Resource-Pack/Website/master/image/background/transparency_16x.png" />
+        </div>
+      </div>
+    </v-list>
+
+    <texture-modal
+      :opened="modalOpen"
+      :closeModal="closeModal"
+      :textureID="modalTextureID"
+      :textureObj="modalTextureObj"
+      :contributors="displayed.contributors"
+    ></texture-modal>
+
+    <v-btn icon large @click="toTop" v-show="scrollY > 300" class="go_up_btn">
```

Comment:
Pleeeeeease make the scroll bar come back, it's horrible, try to use another css selector to hide it on more specific components

Reply:
It's not only related to the gallery, it's global, it doesn't affect this pull request